omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/field/ConceptFragmentController.java

```diff
@@ -0,0 +1,41 @@
+package org.openmrs.module.registrationapp.fragment.controller.field;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+import org.openmrs.Concept;
+import org.openmrs.ConceptAnswer;
+import org.openmrs.api.context.Context;
+import org.openmrs.ui.framework.annotation.FragmentParam;
+import org.openmrs.ui.framework.fragment.FragmentModel;
+import org.openmrs.util.LocaleUtility;
+
+public class ConceptFragmentController {
+
+	public void controller(FragmentModel model, @FragmentParam(required = true, value = "conceptId") Integer conceptId) throws Exception {
```

Comment:
The UI framework should automatically do for you this type conversion from conceptId to Concept object, just change the argument type to Concept